src/EnergyPlus/IntegratedHeatPump.cc

```diff
@@ -1230,6 +1230,7 @@ void GetIHPInput(EnergyPlusData &state)
 
         ihp.SCCoilType = "COIL:COOLING:DX:VARIABLESPEED";
         ihp.SCCoilName = AlphArray(3);
+        ihp.SCCoilTypeNum = DataLoopNode::ConnectionObjectType::CoilCoolingDXVariableSpeed;
```

Comment:
Need enums for calls to compsets. Should address earlier comment about naming for enums.